resources/istio/templates/rbac-migration-remove-dr.yaml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Please create a follow up for us to clean this after next release :)

Reply:
Followup created: https://github.com/kyma-project/kyma/issues/8066